lib/ansible/module_utils/docker_common.py

```diff
@@ -35,22 +32,23 @@
 try:
     from requests.exceptions import SSLError
     from docker import __version__ as docker_version
-    from docker.errors import APIError, TLSParameterError, NotFound
+    from docker.errors import APIError, TLSParameterError
+    # from docker.errors import NotFound
```

Comment:
Ah, why not delete these codes?

Reply:
Good point. I think I first commented them out to see whether CI breaks anything. Since it didn't, I've removed them now.